### PR TITLE
Adding the PHP challenge

### PR DESCRIPTION
**Pull request by frifri**

### CODE REVIEW

File: web-team/php-challenges.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# PHP challenge #
+
+We want to create a database of all the PAX related guide currently available on Guidebook for a case study.
```
Comment (cooncesean) [inline web-team/php-challenges.md]: Spelling: `guide` => `guides` ?

File: web-team/php-challenges.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+**Challenge:** Create a PHP(5) project that will retrieve a list of guides using the `pax` keyword and save it into a database.
```
Comment (cooncesean) [inline web-team/php-challenges.md]: Spelling: `save it` => `insert them`.

File: web-team/php-challenges.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+The URL used to search for guides is `https://guidebook.com/om/service/v2/search/`. You will have to send a GET parameter `q` with the keyword you want to use to do your research.
+
+a guide object will look like:
```
Comment (cooncesean) [inline web-team/php-challenges.md]: Spelling `a guide` => `A guide`.

**Comment by cooncesean** [comment]
This looks reasonable but you may want to clarify what exactly you're looking for in the solution:
- Are you looking to see if they use PHP's OO patterns to fetch and manipulate the data?
- Are you more interested in how they structure and store the data?
- Do you simply want to see that they know how to write PHP code?

Think about how you'd use the challenge as a jumping off point to discuss the candidates solution and the approach they took.

**Comment by frifri** [comment]
I mostly want to make sure that he/she is using OO patterns properly and see how he/she is structuring the project

**Comment by cooncesean** [comment]
Can you add that in the in challenge markdown -- just to provide some guidance for the candidate:

``` markdown
### What We'll Be Looking At

We want to make sure that the solution implements OO patterns properly and that the project is structured in a logical manner.
```

**Pull request merged**